.travis.yml

```diff
@@ -7,11 +7,11 @@ dist: xenial
 addons:
   homebrew:
     packages:
-      - ninja
+      - meson
```

Comment:
I assume ninja gets pulled in as a dependency of meson?  I'd be slightly more comfortable with it explicitly listed I think, but no big deal.

Reply:
yup it's a dependency of meson